### ISSUE TITLE
Has this been deprecated in favor of https://github.com/Nordstrom/serverless-artillery ?

### ISSUE THREAD

**Issue by ryangardner** [Nordstrom/artillery-lambda#1]
I didn't see this one in npm, but did see `serverless-artillery` in NPM and it looks like both are done by Nordstrom - from looking at it, the `serverless-artillery` one certainly _seems_ like it's the more active one at the moment.

**Comment by RobGruhl** [comment]
Yes, I'd recommend looking at serverless-artillery and the serverless-artillery-workshop project.  I'm checking with the authors on this one.

**Comment by ryangardner** [comment]
Yep. Already looked into those. They are working really well so far. Thanks!
